File: tmdbapi/src/main/java/com/afterroot/tmdbapi/model/config/ConfigResults.java
package com.afterroot.tmdbapi.model.config;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;
import java.util.List;

import com.afterroot.tmdbapi.model.core.AbstractJsonMapping;


public class ConfigResults extends AbstractJsonMapping implements Serializable {

    @JsonProperty("images")
    private TmdbConfiguration tmdbConfiguration;
    @JsonProperty("change_keys")
    private List<String> changeKeys;


    public TmdbConfiguration getTmdbConfiguration() {
        return tmdbConfiguration;
    }


    public void setTmdbConfiguration(TmdbConfiguration tmdbConfiguration) {
        this.tmdbConfiguration = tmdbConfiguration;
    }


    public List<String> getChangeKeys() {
        return changeKeys;
    }


    public void setChangeKeys(List<String> changeKeys) {
        this.changeKeys = changeKeys;
    }
}

File: tmdbapi/src/main/java/com/afterroot/tmdbapi/model/config/ConfigResults.kt
package com.afterroot.tmdbapi.model.config

import com.afterroot.tmdbapi.model.core.AbstractJsonMapping
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

class ConfigResults : AbstractJsonMapping(), Serializable {
    @JsonProperty("images")
    var imagesConfig: ImagesConfig? = null

    @JsonProperty("change_keys")
    var changeKeys: List<String>? = null
}